app/assets/js/script.js

```diff
@@ -4,6 +4,11 @@
   var $win = $(window);
   var $doc = $(document);
 
+  // For assetgraph to pick up strings as urls
```

Comment:
I don't really understand what you mean by this. Does AG parse it or something looking for this specific method?